runtime/autoload/man.vim

```diff
@@ -75,7 +75,13 @@ function man#get_page(...) abort
 
   setlocal modifiable
   silent keepjumps norm! 1G"_dG
-  let $MANWIDTH = winwidth(0)
+  if empty($MANWIDTH)
+    if &textwidth
+      let $MANWIDTH = &textwidth
+    else
+      let $MANWIDTH = winwidth(0)
+    endif
```

Comment:
Sorry my vimscript foo is weak, I've had a skim through `usr_41`, what are the `?` and `:` doing here?